packages/lesswrong/components/posts/PostsPage/PostBodyPrefix.tsx

```diff
@@ -53,7 +53,7 @@ const PostBodyPrefix = ({post, query, classes}: {
   const currentUser = useCurrentUser();
 
   return <>
-    {reviewIsActive() && postEligibleForReview(post) && <div className={classes.reviewVoting}>
+    {reviewIsActive() && postEligibleForReview(post) && postIsVoteable(post) && <div className={classes.reviewVoting}>
```

Comment:
Makes it so the Review Header is hidden during the Review Phase if the post didn't get at least 1 positive vote

Reply:
🙏